src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -580,6 +592,12 @@ private JsonObject findAccountForFeeFineAction(JsonObject feeFineAction) {
     return accountsClient.get(UUID.fromString(feeFineAction.getString("accountId"))).getJson();
   }
 
+  private JsonObject scaleFields(JsonObject feeFineAction, String fieldOne, String fieldTwo) {
+    return feeFineAction
+      .put(fieldOne, new BigDecimal(feeFineAction.getDouble(fieldOne)).setScale(2))
+      .put(fieldTwo, new BigDecimal(feeFineAction.getDouble(fieldTwo)).setScale(2));
+  }
+
```

Comment:
I don't think we need this method at all

Reply:
removed